src/App.tsx

```diff
@@ -1,61 +1,112 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
```

Comment:
you have to define this type yourself

Reply:
Sorry, my bad, I accidentally sent only App.tsx.